pkg/routeagent-driver/handlers/kubeproxy-iptables/kp-iptables.go

```diff
@@ -0,0 +1,100 @@
+package kp_iptables
+
+import (
+	"k8s.io/klog"
+
+	"github.com/submariner-io/admiral/pkg/log"
+	k8sV1 "k8s.io/api/core/v1"
+
+	submV1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	"github.com/submariner-io/submariner/pkg/event"
+)
+
+type SyncHandler struct {
+	event.HandlerBase
+	clusterID        string
+	localClusterCidr []string
+	localServiceCidr []string
+
+	cniIface *cniInterface
+}
+
+func NewSyncHandler() *SyncHandler {
+	return &SyncHandler{}
+}
+
+func (kp *SyncHandler) GetName() string {
+	return "kubeproxy-iptables-handler"
+}
+
+func (kp *SyncHandler) GetNetworkPlugin() string {
+	return event.AnyNetworkPlugin
+}
+
+func (kp *SyncHandler) Init() error {
+	return nil
+}
+
+func (kp *SyncHandler) TransitionToNonGateway() error {
```

Comment:
You don't need to add the logging again, just add the logging handler during initialization of your controller., see: https://github.com/submariner-io/submariner/blob/f000f3605e05f334623c7227c9827c4e920c9439/pkg/networkplugin-syncer/main.go#L30

Reply:
Yes, I will change the log messages in further PRs. This was just for my debugging until I handle the corresponding events.